scenarios/openshift/detach_ebs_volumes.yaml

```diff
@@ -0,0 +1,9 @@
+aws_access_key_id:                            # only if system-wide credentials are not
+aws_secret_access_key:                        # configured or using different credentials
```

Comment:
We might want to introduce a cloud_type variable which is set to aws in this case as we will be adding support for Azure and GCP as well for this scenario. For example: https://github.com/krkn-chaos/krkn/blob/main/scenarios/openshift/aws_node_scenarios.yml#L10

Reply:
The goal is to use the same config and load things differently based on the cloud_type.